2020-06-22 13:15:57,564 - __main__ - WARNING - warning
2020-06-22 13:15:57,569 - __main__ - CRITICAL - fatal
2020-06-22 13:15:57,569 - __main__ - CRITICAL - critical
2020-06-22 13:15:57,569 - __main__ - ERROR - error
2020-06-22 13:16:07,735 - __main__ - WARNING - warning
2020-06-22 13:16:07,735 - __main__ - CRITICAL - fatal
2020-06-22 13:16:07,735 - __main__ - CRITICAL - critical
2020-06-22 13:16:07,736 - __main__ - ERROR - error
2020-06-22 13:16:07,736 - __main__ - ERROR - Failed to open sklearn.txt from logger.exception
Traceback (most recent call last):
  File "D:/MySpace/Python/bulitin/s_logging.py", line 49, in <module>
    open("../data/mysql.inis", "rb")
FileNotFoundError: [Errno 2] No such file or directory: '../data/mysql.inis'
